README.md

```diff
@@ -418,6 +418,21 @@ with d:
 <div id="header"></div>
 ```
 
+And text nodes can be added with the `dominate.util.text` function:
+
+```python
+from dominate.util import text
+para = p(__pretty=False)
```

Comment:
I suggest paragraph as variable name, because it is more explicit and clear to new users. Could at first sight also be interpreted as parameter. Otherwise looks good to me.